### PR TITLE
Standardize

### PR DESCRIPTION
**Pull request by liveh2o**
Replace Mad Rubocop with Standard to follow Ruby community styles. Fix rule violations, and add a lint job to the CI workflow that checks for violations on all pull requests.

### CODE REVIEW

File: lib/active_remote/association.rb [hunk 1]
```diff
@@ -131,10 +131,10 @@ def validate_scoped_attributes(associated_class, object_class, options)
         raise "Could not find attribute: '#{options[:scope]}' on #{associated_class}" unless associated_class.public_instance_methods.include?(options[:scope])
       end
 
-    private
```
Comment (film42) [inline lib/active_remote/association.rb:134]: @mmmries would like a word with you. 😂

**Review by film42 — APPROVED** [review]
Looks good to me. Happy to see this codebase get a little refresh in style.

File: lib/active_remote/association.rb (hunk 1 above)
Comment (mmmries) [reply lib/active_remote/association.rb:134]: 👋 hi friends

Comment (mmmries) [reply lib/active_remote/association.rb:134]: I've left the mad rubocop behind. I'm all about that zen rubocop now

Comment (liveh2o) [reply lib/active_remote/association.rb:134]: Hahahaha

**Pull request merged**